src/libstd/fs/mod.rs

```diff
@@ -361,6 +436,15 @@ impl OpenOptions {
     ///
     /// This option indicates whether a new file will be created if the file
     /// does not yet already exist.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
i changed this for all of the examples which aren't wrapped in a function: those ones are already effectively no_run, as the function is never called